pkg/memio/ioport_linux_386.s

```diff
@@ -0,0 +1,62 @@
+// Copyright 2012-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+	// From go runtime compiler source:
+        //TODO: INB DX, AL                      // ec
+        //TODO: INW DX, AX                      // 66ed
+        //TODO: INL DX, AX                      // ed
+
+TEXT memio·archInb(SB),$0-8
+        MOVL    port+0(FP), DX
+	BYTE $0xec
+	//INB	DX, AL
```

Comment:
Isn't this instruction the whole point?

Reply:
I see, the assembler doesn't support this instruction. To make it clearer, can you write the line as:

```
BYTE $0xec  // INB DX, AL
```